include/bouncer.h

```diff
@@ -118,6 +128,8 @@ extern int cf_sbuf_len;
 #include "janitor.h"
 #include "hba.h"
 #include "pam.h"
+#include "messages.h"
```

Comment:
keep alphabetical order

Reply:
I moved it before `pam.h` now to make it bit more alphabetical. But this list is currently not in alphabetical order at all.